src/Symfony/Component/Messenger/Middleware/ValidatingMiddleware.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Middleware;
+
+use Symfony\Component\Messenger\Exception\ValidationFailedException;
+use Symfony\Component\Messenger\MiddlewareInterface;
+use Symfony\Component\Messenger\ValidatedMessageInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+/**
+ * @author Magnus Nordlander <magnus@fervo.se>
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class ValidatingMiddleware implements MiddlewareInterface
```

Comment:
`ValidationMiddleware` feels better to me and is more coherent with the other middleware names 🤔 (except `LoggingMiddleware` 🙊)